src/components/Tabs.react.js

```diff
@@ -168,6 +258,7 @@ export default class Tabs extends Component {
     render() {
         let EnhancedTabs;
```

Comment:
Not a change from this PR, but consider renaming to `enhancedTabs` - Pascal-Casing for variables deviates from the norm.